src/components/Messages/index.tsx

```diff
@@ -12,7 +12,7 @@ import PreviewList from './PreviewList';
 const Messages: NextPage = () => {
   const currentProfile = useAppStore((state) => state.currentProfile);
 
-  if (!currentProfile || !isFeatureEnabled('messages', currentProfile.id)) {
+  if (!isFeatureEnabled('messages', currentProfile?.id)) {
```

Comment:
Last one, but same as above. Please lmk if you disagree or if I'm missing something, but I think it's just safer to explicitly check.

Reply:
+1 we need this @bhavya2611 🙏🏼